### PR TITLE
[WIP] mkbundle commands for cross compilation.

### PR DESCRIPTION
**Pull request by migueldeicaza**

### CODE REVIEW

File: man/mkbundle.1 [hunk 1]
```diff
@@ -21,9 +21,34 @@ default only the assemblies specified in the command line will be
...
+	$ mkbundle --local-targets	
+	Available targets:
+		default	- Current System Mono
+		4.4.0-macosx-x86
+		4.4.0-debian-8-arm64
+	$ mkbundle --cross 4.4.0-debian-8-x86 hello.exe -o hello-debian
+.fi
+.PP
+The above will bundle your native library into hello-debian for
+PowerPC running on ARM64.
```
Comment (akoeplinger) [inline man/mkbundle.1]: "PowerPC running on ARM64" <-- that doesn't make much sense.

File: mcs/tools/mkbundle/mkbundle.cs [hunk 2]
```diff
@@ -46,9 +47,12 @@ class MakeBundle {
 	static bool skip_scan;
 	static string ctor_func;
 	static bool quiet;
+	static string cross_target = null;
+	static string fetch_target = null;
 	static bool custom_mode = true;
 	static string embedded_options = null;
 	static string runtime = null;
+	static string target_server = "https://runtimes.go-mono.com/raw/";
```
Comment (akoeplinger) [inline mcs/tools/mkbundle/mkbundle.cs]: The SSL certificate for that domain is invalid.

Why aren't we using download.mono-project.com for this?

**Comment by HinTak** [comment]
https://runtimes.go-mono.com/raw/ does not resolve...

**Comment by HinTak** [comment]
https://github.com/mono/mono/pull/3000/commits/0447d4b751573d13d1d358b2741e49c34914de98 changes the url to https://download.mono-project.com/runtimes/raw/ , but it is 404 at the moment:

The requested URL /runtimes/raw/ was not found on this server.

**Comment by HinTak** [comment]
It is all working smoothly as of 4.8 I just tried. :-)

**Pull request merged**